packages/modules/vehicles/vwid/libvwid.py

```diff
@@ -41,14 +41,18 @@ def password_form(self, text):
                 text = text[text.find('\n'):text.rfind('\n')].strip()
 
                 for line in text.split('\n'):
-                    (name, val) = line.strip().split(':', 1)
-                    val = val.strip('\', ')
-                    objects[name] = val
+                    try:
+                        (name, val) = line.strip().split(':', 1)
+                    except Exception:
+                        self.log.debug("vwid.libvwid.password_form: skip line: (" + line + ")")
+                    else:
+                        val = val.strip('\', ')
+                        objects[name] = val
 
         json_model = json.loads(objects['templateModel'])
 
         if ('errorCode' in json_model):
-            self.log.error("Login error: %s", json_model['errorCode'])
+            self.log.debug("vwid.libvwid: Login error: %s", json_model['errorCode'])
```

Comment:
Das Python-Logging ist so konfiguriert, dass das Modul, aus dem die Logmeldung stammt, und das zugehörige Fahrzeug in der Lognachricht enthalten sind.
```suggestion
            self.log.error("Login error: %s", json_model['errorCode'])
```
Die vorherige Einstufung in Error war richtig, da es sich hier ja um einen Fehler handelt, sollte er auch als solcher geloggt werden.